Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN cd /opt && \
 # and hosts.
 RUN yum install -y nss_wrapper && yum clean all
 
+RUN yum install -y numpy && yum clean all
```

Comment:
we should probably combine this with the yum call on the previous to help reduce the number of layers produced